macOSUpgrade.sh

```diff
@@ -153,10 +153,37 @@ osinstallersetupdDaemonSettingsFilePath="/Library/LaunchDaemons/com.jamfps.clean
 ##Launch agent settings for filevault authenticated reboots
 osinstallersetupdAgentSettingsFilePath="/Library/LaunchAgents/com.apple.install.osinstallersetupd.plist"
 
+##Amount of time (in seconds) to allow a user to connect to AC power before moving on
+acPowerWaitTimer="120"
```

Comment:
I think it would be better if there was an option to cancel to allow user such kind of opportunity.
Someone may think it is too short or long to connect my mac to AC. It would be so boring how much time was valid for acPowerWaitTimer. It should not be a static value. It should be given as an user parameter.

Reply:
It could be passed as a parameter, with a default value of null or 0. If null or 0, then don't give the option, if not null and above 0, then give the opportunity to connect to AC power.

Reply:
It sounds good if its default value is 0. If it is not given, set acPowerWaitTimer to 0 as default.
It would be nice if there is a validation for makeing sure it is a positive number of 0 or more.

Reply:
Check [6ec35f1](https://github.com/kc9wwh/macOSUpgrade/pull/115/commits/6ec35f14363fa9fb9fd8e6c008e894ffe13b1309) for this.